sql-join/20-5-24 EX 8.sql

```diff
@@ -0,0 +1,8 @@
+use telecom;
+
+SELECT et.id, location, fault_severity, event_type, severity_type, resource_type, log_feature, volume

```

Comment:
I think we talked about this in class, but the order of tables/columns should be train, event_type, severity_type, resource_type, log_feature and the key to join on is "id" (not your fault since this wasn't made clear in the instructions).